pallets/parachain-staking/src/inflation.rs

```diff
@@ -25,9 +25,13 @@ use sp_runtime::{Perbill, RuntimeDebug};
 use substrate_fixed::transcendental::pow as floatpow;
 use substrate_fixed::types::I64F64;
 
+// Milliseconds per year
+const MS_PER_YEAR: u64 = 31_557_600_000;
+
 fn rounds_per_year<T: Config>() -> u32 {
-	let blocks_per_round = <Pallet<T>>::round().length;
-	T::SlotsPerYear::get() / blocks_per_round
+	let blocks_per_round = <Pallet<T>>::round().length as u64;
+	let blocks_per_year = MS_PER_YEAR / T::BlockTime::get();
```

Comment:
We could probably have a new helper function named `blocks_per_year` and replace `MS_PER_YEAR / T::BlockTime::get()` by it everywhere.